matrix-e2e.c

```diff
@@ -298,6 +304,118 @@ static int matrix_restore_e2e_account(MatrixConnectionData *conn)
     return ret;
 }
 
+/* TODO: Check if we need this every 10mins or so */
```

Comment:
ftr `/sync` now sends back the number of one-time keys, so you can hang it off that rather than a timer.

Reply:
Done; now I read the device_one_time_keys_count member and use it to decide whether to send keys.